lib/ex_aws_credential_process.ex

```diff
@@ -40,7 +40,13 @@ defmodule ExAwsCredentialProcess do
       raise ArgumentError, "no refresh strategy was provided"
     end
 
-    :ets.new(@table_name, [:set, :named_table, :protected, read_concurrency: true])
+    :ets.new(@table_name, [
+      :set,
+      :named_table,
+      table_permission(Mix.env()),
```

Comment:
for :protected, the owner process is the only one that can write to it which was a problem in test...